src/media/js/addon/components/addon.js

```diff
@@ -55,6 +61,30 @@ export class Addon extends React.Component {
     }
     return <h2>{this.props.name}</h2>
   }
+
+  isButtonDisabled() {
+    return (this.props.isPublishing ||
+            this.props.isRejecting);
+  }
+
+  publishButton() {
```

Comment:
i don't think this needs to be refactored into its own method yet, but if we are, i like to prefix methods that return components with ```render``` (e.g., renderPublishButton)